lib/config/app_config.dart

```diff
@@ -90,6 +96,25 @@ abstract class AppConfig {
   static String? issueId;
 
   static void loadFromJson(Map<String, dynamic> json) {
+    if (json['homeserver'] != null && json['homeserver'] is String) {
+      if (json['homeserver'] != '') {
+        bubbleSizeFactor = json['homeserver'];
+      }
+    }
+
+    if (json['register_site'] != null && json['registration_url'] is String) {
+      if (json['registration_url'] != '') {
+        registrationUrl = json['registration_url'];
+      }
+    }
+
+    if (json['twake_workplace_homeserver'] != null &&
+        json['twake_workplace_homeserver'] is String) {
+      if (json['twake_workplace_homeserver'] != '') {
+        twakeWorkplaceHomeserver = json['twake_workplace_homeserver'];
+      }
```

Comment:
```suggestion
    if (json['twake_workplace_homeserver'] != null &&
        json['twake_workplace_homeserver'] is String && 
        json['twake_workplace_homeserver'].isNotEmpty) {

        twakeWorkplaceHomeserver = json['twake_workplace_homeserver'];
```

Reply:
json['twake_workplace_homeserver'] is dynamic, can't use .isNotEmpty

Reply:
okey just use one if is okey